### PR TITLE
[SW-676] Truncate long names in vault creation review

### PR DESCRIPTION
**Pull request by peronczyk**

### CODE REVIEW

File: src/popup/components/MultisigVaultCreateReview.vue [hunk 1]
```diff
@@ -114,6 +121,7 @@ import TokenAmount from './TokenAmount.vue';
 import FormSelect from './form/FormSelect.vue';
 
 import LoadingIcon from '../../icons/animated-spinner.svg?skip-optimize';
+import Truncate from './Truncate.vue';
```
Comment (pstachel) [inline src/popup/components/MultisigVaultCreateReview.vue:124]: please group the imports

File: src/popup/components/Truncate.vue [hunk 2]
```diff
@@ -18,49 +24,45 @@
...
 
-          if (scrollElem && containerElem) {
-            const { scrollWidth, offsetWidth } = containerElem;
+    onMounted(() => {
+      watch(
+        () => props.str,
+        () => {
+          shouldScroll.value = false;
+
+          if (props.fixed) return;
```
Comment (pstachel) [inline src/popup/components/Truncate.vue:59]:
```suggestion
          if (props.fixed) {
              return;
          }    
```

**Review by pstachel — APPROVED** [review]

File: src/popup/components/Truncate.vue (hunk 2 above)
Comment (peronczyk) [reply src/popup/components/Truncate.vue:59]: Done

File: src/popup/components/Truncate.vue [hunk 3]
```diff
@@ -6,10 +6,16 @@
...
+        fixed,
+        scrollable: shouldScroll,
+      }"
       :style="cssVars"
     >
-      <span ref="scroll">{{ nameComponent || str }}</span>
+      <span
+        ref="scroll"
+        class="inner"
+      >{{ nameComponent || str }}</span>
```
[the remark this answers is not in the document]
Comment (peronczyk) [reply src/popup/components/Truncate.vue:18]: Done with the use of `v-text`.

File: src/popup/components/MultisigVaultCreateReview.vue (hunk 1 above)
Comment (peronczyk) [reply src/popup/components/MultisigVaultCreateReview.vue]: Sure, done.

**Review by konradodo — APPROVED** [review]

**Review by ifaouibadi — APPROVED** [review]

**Pull request merged**